src/components/Login.tsx

```diff
@@ -51,71 +39,30 @@ const Login = (props: Props) => {
     setLastName(value);
   }
 
-  const generateAgent = async (event: React.SyntheticEvent) => {
-    setLoading(true);
-    let agentStatus = await ad4mClient.agent.generate(password);
-    const agentPerspective = await ad4mClient.perspective.add(
-      "Agent Profile"
-    );
-    const links = [];
-
-    if (firstName) {
-      const link = await ad4mClient.perspective.addLink(
-        agentPerspective.uuid,
-        new Link({
-          source: SOURCE_PROFILE,
-          target: firstName,
-          predicate: PREDICATE_FIRSTNAME
-        })
-      );
-
-      links.push(link);
-    }
-
-    if (lastName) {
-      const link = await ad4mClient.perspective.addLink(
-        agentPerspective.uuid,
-        new Link({
-          source: SOURCE_PROFILE,
-          target: lastName,
-          predicate: PREDICATE_LASTNAME
-        })
-      );
-
-      links.push(link)
-    }
-
-    const cleanedLinks = [];
-
-    for (const link of links) {
-      const newLink = JSON.parse(JSON.stringify(link));
-      newLink.__typename = undefined;
-      newLink.data.__typename = undefined;
-      newLink.proof.__typename = undefined;
-
-      cleanedLinks.push(newLink);
-    }
-
-    await ad4mClient.agent.updatePublicPerspective({
-      links: cleanedLinks
-    })
-
-    props.handleLogin(agentStatus.isUnlocked, agentStatus.did!);
-
-    console.log("agent status in generate: ", agentStatus);
-  };
-
-  const unlockAgent = async (event: React.SyntheticEvent) => {
-    setLoading(true);
-    let agentStatus = await ad4mClient.agent.unlock(password);
-    props.handleLogin(agentStatus.isUnlocked, agentStatus.did!);
-
-    console.log("agent status in unlock: ", agentStatus);
-  }
-
   return (
     <div>
       <Stack align="center" spacing="xl">
+        <div style={{
+          position: 'fixed',
+          top: 20,
+          right: 20
+        }}>
+          <Tooltip
+            label="Change ad4m-executor url"
```

Comment:
What's this used for? do we have the function in this PR to set customized ad4m-host url?

Reply:
yes